### DISCUSSION TITLE
wire:navigate on wire:click? wire:click.prefetch no longer available since v3

### DISCUSSION THREAD

**Discussion by codevApp** [livewire/livewire#7938] [1. Help]
Hello everyone,
wire:navigate works great.
But since wire:click.prefetch is no longer possible in v3, I wonder how to cover the behavior there?
everything with wire:navigate is super fast. But if you use pagination or filters within a component, for example, you don't have the same option as with wire:navigate?

**Comment by danie-ramdhani** [comment]
idk what do you want to achieve, but i read the title and you want to tried to navigate using button click? then you can do this:

```blade
<button x-on:click="Livewire.navigate('/new/url')">Navigate</button>
```

**Comment by shubdragon** [comment]
do you want to prefetch the page on hover while using wire navigate  ,right?

`<a href="/page" wire:navigate.hover>Page</a>`

add `.hover` on `wire:naviagte`

**Comment by codevApp** [comment]
I mean for example the click to the component itself. For example the pagination [https://livewire.laravel.com/docs/pagination#sample-pagination-view](url)

`<button wire:click="previousPage" wire:loading.attr="disabled" rel="prev">Previous</button>`

or something else to a method in the component:

`<button wire:click="setSortBy()"  ..`
